<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_INFO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>751</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>系统版本号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>61</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU型号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_SystemVersion">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CPUType">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>561</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>751</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>启动时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>运行时长：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>主机名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_hostname">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>ChenQianhe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_boottime">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2022.3.1 00:00:10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_runningtime">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>00：00</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_process">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>751</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pid">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="killButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill</string>
      </property>
     </widget>
     <widget class="QLabel" name="pidInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>741</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="path">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="runButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>使用率</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="CPUhorizontalLayout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>341</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="MEMhorizontalLayout_2"/>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>300</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>MEM</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU使用率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>内存使用率：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shutdown">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关   机</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CurrentTime">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_RAM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_rate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>pushButton_shutdown</zorder>
   <zorder>label_CurrentTime</zorder>
   <zorder>progressBar_RAM</zorder>
   <zorder>tabWidget_INFO</zorder>
   <zorder>cpu_rate</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
